rox_navigation/configs/navigation_omni.yaml

```diff
@@ -118,7 +118,10 @@ bt_navigator:
     robot_base_frame: base_link
     odom_topic: /odom
     bt_loop_duration: 10
-    default_nav_to_pose_bt_xml: "$(find-pkg-share rox_navigation)/configs/behavior_trees/navigate_to_pose_w_replanning_and_recovery.xml"
+    # Strict following (Not Working with RViz Nav2 goal)
+    # default_nav_to_pose_bt_xml: "$(find-pkg-share rox_navigation)/configs/behavior_trees/navigate_on_route_graph_w_recovery.xml"
+    # Less strict
+    default_nav_to_pose_bt_xml: "$(find-pkg-share rox_navigation)/configs/behavior_trees/navigate_w_routing_global_planning_and_control_w_recovery.xml"
```

Comment:
routing will not be the default. This has to be set in the launch argument. 

Maybe neo_nav2_bringup needs an update to handle this?